src/moj/components/banner/template.njk

```diff
@@ -1,4 +1,26 @@
-<div class="moj-banner{% if params.type == 'success' %} moj-banner--success{% elif params.type == 'warning' %} moj-banner--warning{% else %} {{- ' ' + params.classes if params.classes}}{% endif %}" {%- for attribute, value in params.attributes %} {{ attribute }}="{{ value }}"{% endfor %} role="region" aria-label="{{ params.iconFallbackText | default(params.type) | default("banner") }}">
+{#- Set classes for this component #}
+{%- set classNames = ["moj-banner"] -%}
+
+{%- if params.type == 'success' %}
+  {%- set classNames = classNames.concat("moj-banner--success") %}
+{%- elif params.type == 'warning' %}
+  {%- set classNames = classNames.concat("moj-banner--warning") %}
+{%- elif params.type == 'information' %}
+  {%- set classNames = classNames.concat("moj-banner--information") %}
+{%- endif %}
+
+{%- set classNames = classNames.concat(params.classes) if params.classes %}
```

Comment:
```suggestion
{%- if params.classes %}
{%- set classNames = classNames.concat(params.classes)  %}
{%- endif %}
```
This solves the issue.  I think if `params.classes` was empty it was still running the `set`. Wrapping it in the if, avoids this.

Reply:
Thanks -- I had just figured that out too! Seems to be working my end from what I can see :) Thank you for your review and comments Chris

Reply:
No worries.  Thanks for the PR.

If you create another PR in the future, the actions will run automatically so you'll see any issues sooner.  It's just because you're a first time contributor that they require permission to run.